source/globals.d.ts

```diff
@@ -70,9 +67,10 @@ interface HTMLFormControlsCollection {
 	[key: string]: HTMLInputElement | HTMLTextAreaElement | HTMLButtonElement;
 }
 
-declare module '*.svg' {
-	const Icon = (): JSX.Element => JSX.Element;
-	export default Icon;
+declare module 'react' {
+	const FC = (): JSX.Element => JSX.Element;
+	const React = {FC};
+	export default React;
```

Comment:
I think this isn't needed, right? Doesn't the module already export the correct types?

Reply:
In the `react` module, the return type of functional components is `ReactElement`, not `JSX.Element`. So when I remove the type declaration, I get errors like this:

```
source/github-widgets/notice-bar.tsx:17:6 - error TS2786: 'XIcon' cannot be used as a JSX component.
  Its return type 'ReactElement<any, any> | null' is not a valid JSX element.
    Type 'ReactElement<any, any>' is not assignable to type 'Element'.

17                      <XIcon/>
```